tests/data/example.bam

```diff
@@ -0,0 +1 @@
+foobar
```

Comment:
TODO for Tom: Create a ticket to explore how to deal with testing larger files in this package